.codecov.yml

```
@@ -1,4 +1,4 @@
 ignore:
   - "src/examples/.*"
   - "src/tutorials/.*"
-  - "src/helpers/exports/.*" # until we figure out how to check asy runs - exclude.
\ No newline at end of file
+  - "src/helpers/exports/.*" # exclode exports, since there are no CI tests for Asy-exports
```

```suggestion
  - "src/helpers/exports/.*" # exclude exports, since there are no CI tests for Asy-exports
```
Maybe it's time for Vale checks of .yml comments :smile: